snmp/datadog_checks/snmp/data/default_profiles/ruckus-unleashed.yaml

```
@@ -0,0 +1,78 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - ruckus.yaml
```

```suggestion
  - ruckus.yaml
```

Since we are extending `ruckus.yaml` and it already contains `_base.yaml` and `_generic-if.yaml`, you don't need to have then here.